sql/step_73_operation_report_button.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO MENUITEM VALUES ('operationlist','angal.menu.btn.operationlist','angal.menu.operationlist','x','O','printing','org.isf.stat.gui.OperationsListLauncher','N',3);

```

Comment:
There is already a step_73 (`step_73_ward_visit_duration.sql`)

This should be renamed to `step_74_operation_report_button.sql` and then added to the bottom of the file: `step_04_all_folling_steps.sql`

Reply:
I fixed the commit. Thanks @dbmalkovsky